pkg/binlog/common/replication.go

```diff
@@ -28,10 +28,14 @@ var (
 )
 
 // SetDefaultReplicationCfg sets some default value for BinlogSyncerConfig
-func SetDefaultReplicationCfg(cfg *replication.BinlogSyncerConfig) {
+func SetDefaultReplicationCfg(cfg *replication.BinlogSyncerConfig, reConnect bool) {
```

Comment:
How about replcaing `reConnect` with `RetryCount`?

rest LGTM.

Reply:
update in e856f66